packages/nimble-components/src/select/styles.ts

```diff
@@ -25,6 +25,11 @@ export const styles = css`
         /* We are using flex `order` to define the visual ordering of the selected value,
          error icon, and dropdown arrow because they are not "interactive" i.e. part of the tab order */ ''
     }
+
+    [part='selected-value'].placeholder {
+        color: ${placeholderFontColor};
+    }
+
     [part='selected-value'] {
         order: 1;
     }
```

Comment:
nit / pref. We already have a class on that item in the template. Using that is a bit cleaner
```suggestion
    .selected-value.placeholder {
        color: ${placeholderFontColor};
    }

    .selected-value {
        order: 1;
    }
```